### ISSUE TITLE
Blocks of type "rate" are not expected here

### ISSUE THREAD

**Issue by okkevdeijkGV** [labd/terraform-provider-commercetools#91]
Hi all,

I am trying to use the following TF file with a `commercetools_tax_category`
```
resource "commercetools_tax_category" "Medicinal" {
  name = "Medicinal"
  key =  "Medicinal"
  rate {
    name = "Medicinal"
    amount = 0.04
    included_in_price = true
    country = "IO"
  }
}

resource "commercetools_tax_category" "standard" {
  name = "standard"
  key =  "standard"
  rate {
    name = "standard"
    amount = 0.22
    included_in_price = true
    country = "IO"
  }
  rate {
    name = "standard-gb"
    amount = 0.23
    included_in_price = true
    country = "GB"
  }
}
```

Unfortunately I am bumping in to the following error:

```
Error: Unsupported block type

  on tax-categories.tf line 4, in resource "commercetools_tax_category" "Medicinal":
   4:   rate {

Blocks of type "rate" are not expected here.


Error: Unsupported block type

  on tax-categories.tf line 15, in resource "commercetools_tax_category" "standard":
  15:   rate {

Blocks of type "rate" are not expected here.


Error: Unsupported block type

  on tax-categories.tf line 21, in resource "commercetools_tax_category" "standard":
  21:   rate {

Blocks of type "rate" are not expected here.
```

Is there something I am doing wrong or is this a bug/TF versioning issues?

**Comment by davidweterings** [comment]
It seems we forgot to update the documentation for this resource 😓  see https://commercetools-terraform-provider.readthedocs.io/en/latest/resource_tax_category_rate/ on how to add rates to tax categories.

tldr; remove the rates and use the new `commercetools_tax_category_rate` resource to create those.

**Comment by okkevdeijkGV** [comment]
Aha, would you like that I create a PR for it as well ;)? Thanks!

**Comment by davidweterings** [comment]
It would be much appreciated :)!
